web/themes/custom/mytheme/mytheme.info.yml

```diff
@@ -0,0 +1,16 @@
+name: My Theme
+type: theme
+description: This is my first custom theme project
+core: 8.x
+screenshot: mytheme.png
+
```

Comment:
Not sure if the tutorial mentioned this, but there is usually a `base theme` key in the theme.info.yml file. If one isn't specified, it uses a theme called `stable` So the equivalent here would be:

`base theme: stable`

Keep going with what the tutorial specifies, but figured that would be worth noting. I believe a `base theme` value will be required in Drupal 9.